recipes/slamdunk/meta.yaml

```diff
@@ -1,28 +1,27 @@
-{% set version = "0.3.3" %}
-{% set sha256 = "9e8ec79b9f47e2bcf17185e3c14b98039e137140a003400e3101f1cbeefc1162" %}
+{% set version = "0.3.4" %}
+{% set sha256 = "1dddcec91fac6ab199855629bc004e4cc972ce5c66199b70b703f23ab7eab00d" %}
 
 package:
   name: slamdunk
   version: {{ version }}
 
 source:
   sha256: '{{sha256}}'
-  url: "https://github.com/t-neumann/slamdunk/archive/v0.3.3.tar.gz"
+  url: "https://github.com/t-neumann/slamdunk/archive/v{{ version }}.tar.gz"
 
 build:
   entry_points:
     - slamdunk=slamdunk.slamdunk:run
     - alleyoop=slamdunk.alleyoop:run
     - splash=slamdunk.splash:run
-  number: 0
+  number: 1
```

Comment:
This should be 0 for new versions.

Reply:
It's not a new version, but a new build due to the broken openssl dependency.